server/events/comment_parser_test.go

```diff
@@ -707,3 +799,38 @@ var UnlockUsage = "`Usage of unlock:`\n\n ```cmake\n" +
   Arguments or flags are not supported at the moment.
   If you need to unlock a specific project please use the atlantis UI.` +
 	"\n```"
+
+func TestCommentParser_helpComment(t *testing.T) {
+	type fields struct {
+		GithubUser      string
+		GitlabUser      string
+		BitbucketUser   string
+		AzureDevopsUser string
+		ApplyDisabled   bool
+	}
+	type args struct {
+		applyDisabled bool
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		args   args
+		want   string
+	}{
+		// TODO: Add test cases.
```

Comment:
todo?

Reply:
I think this is covered by other tests and the test case is not needed